image/v1/types.go

```diff
@@ -605,6 +605,10 @@ type ImageBlobReferences struct {
 	// not have separate config blobs and this field will be set to nil if so.
 	// +optional
 	Config *string `json:"config" protobuf:"bytes,2,opt,name=config"`
+	// manifests is the list of manifests that compose this image. For a
+	// single architecure image, it is empty. For a multi-arch image, it
+	// consists of single architecture images.
+	Manifests []string `json:"manifests,omitempty" protobuf:"bytes,4,rep,name=manifests"`
```

Comment:
is this field mutually exclusive to any other fields?  Layers comes to mind.

Reply:
maybe, it doesn't matter for clients

Reply:
it can write:

> For a multi-arch image, it consists of single architecture images, and such images shouldn't have layers nor config.

Reply:
The doc now makes sense, but the validation is not yet present in https://github.com/openshift/openshift-apiserver/pull/309/files#diff-4e280f22beaa1bd7ae3747b41a8f5797ad011d86d61adbf82500c73151daeddfR806